src/Slave.php

```diff
@@ -256,7 +256,7 @@ public function getMaxRequests()
     }
 
     /**
-     * @return int|null
+     * @return bool|null
```

Comment:
Can you make this bool? It's never gonna return null?

Reply:
Oh yes, my bad. Good catch.